### PR TITLE
routesrv: log route changes for debugging

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Log route changes for debugging

### CODE REVIEW

File: routesrv/polling.go [hunk 1]
```diff
@@ -109,6 +110,12 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 			}
 			span.SetTag("routes.count", routesCount)
 			span.SetTag("routes.bytes", routesBytes)
+
+			if updated && log.GetLevel() == log.DebugLevel {
```
Comment (RomanZavodskikh) [inline routesrv/polling.go:114]: Maybe `>= log.DebugLevel` here? There is at least one more verbose leve: `TraceLevel`

https://pkg.go.dev/github.com/sirupsen/logrus#pkg-variables

Comment (AlexanderYastrebov) [reply routesrv/polling.go]: Indeed, fixed.

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**